src/components/features/cart/cartItem/CartItem.tsx

```diff
@@ -1,79 +1,68 @@
 import { isValidImageUrl } from '../../../../utils/isValidImageUrl';
+import FallbackImage from '../../../common/fallbackImage/FallbackImage';
 import IconButton from '../../../common/iconButton/IconButton';
 import SelectBox from '../../../common/selectBox/SelectBox';
 import Separator from '../../../common/separator/Separator';
-import { deleteCartItem } from '../api/deleteCartItem';
-import { updateCartItem } from '../api/updateCartItem';
+import useCartMutations from '../hooks/useCartMutations';
 import { CartItemType } from '../types';
 import * as S from './CartItem.styles';
-import defaultImage from '/assets/default_product.png';
 
-interface CartItemProps {
-  cartItem: CartItemType;
+interface CartItemProps extends CartItemType {
   selected: boolean;
   toggle: () => void;
-  onUpdate: () => Promise<void>;
 }
 
-function CartItem({ cartItem, selected, toggle, onUpdate }: CartItemProps) {
+function CartItem({ id, product, quantity, selected, toggle }: CartItemProps) {
+  const { remove, increase, decrease } = useCartMutations(id);
   return (
-    <S.Container data-testid={`CartItem-${cartItem.id}`}>
+    <S.Container data-testid={`CartItem-${id}`}>
       <Separator />
       <S.ActionContainer>
         <SelectBox selected={selected} onClick={toggle} />
         <S.DeleteButton
           onClick={async () => {
-            await deleteCartItem(cartItem.id);
-            onUpdate();
+            remove();
           }}
```

Comment:
async는 붙여줄 필요가 없어보이네요!

```suggestion
          onClick={remove}
```
요로코롬..